package.json

```diff
@@ -8,8 +8,7 @@
     "docs": "node ./scripts/docs/index.js",
     "prepare-release": "git checkout master && git pull --rebase origin master && npm run docs && lerna updated",
     "release": "npm run prepare-release && lerna publish --exact",
-    "format": "prettier \"{packages,scripts}/**/*.js\" --write",
-    "precommit": "lint-staged"
```

Comment:
TODO: add this back, it was blowing up on my Win10 box

Reply:
This looks amazing!

Don't know if it applies but you can ignore the commit hooks if you do `git commit -m "message" -n`.

Reply:
@sto3psl awesome! didn't know I could do that... for context it was only blowing up when i did a commit on windows